test/integration/roles/test_win_package/tasks/main.yml

```diff
@@ -0,0 +1,70 @@
+# test code for the win_package module
+# (c) 2014, Chris Church <chris@ninemoreminutes.com>
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+- name: use win_get_url module to download msi
+  win_get_url:
+    url: "{{msi_url}}"
+    dest: "{{msi_download_path}}"
+  register: win_get_url_result
+
+- name: make sure msi is uninstalled
+  win_package:
+    path: "{{msi_download_path}}"
+    product_id: "{{msi_product_code}}"
+    state: absent
+#  ignore_errors: true
```

Comment:
that's a left over from the win_msi tests, which I think were unreliable.  I agree its confusing as it is though so could be deleted.